src/app/modules/main/module.nim

```diff
@@ -1038,6 +1037,9 @@ method communityLeft*[T](self: Module[T], communityId: string) =
 method communityEdited*[T](
     self: Module[T],
     community: CommunityDto) =
+  if(not self.channelGroupModules.contains(community.id)):
+    echo "main-module, unexisting community key to edit: ", community.id
```

Comment:
maybe it's better to use `warn` from `chronicles`?

Reply:
I'll actually delete the log. It was useful for debugging the crash, but it's not a necessary log, because it's normal for some communities to be edited while not being joined or spectated (in the portal for example)